docs/tutorials/pre_executed/rubin_dp1.ipynb

```diff
@@ -20,15 +20,14 @@
                 "  * How to access Rubin's Data Preview 1 with LSDB\n",
                 "    * on RSP (Rubin Science Platform)\n",
                 "    * on NERSC (National Energy Research Scientific Computing Center) for the [LSST DESC](https://lsstdesc.org) members\n",
+                "    * [Coming soon] on CANFAR (Canadian Advanced Network for Astronomical Research), Canadian IDAC\n",
+                "    * [Coming soon] on LlneA (Laboratório Interinstitucional de e-Astronomia), Brazilian IDAC\n",
```

Comment:
```suggestion
                "    * [Coming soon] on LIneA (Laboratório Interinstitucional de e-Astronomia), Brazilian IDAC\n",
```
I believe this should be capital "I" not lowercase "l", per https://www.linea.org.br/